.editorconfig

```diff
@@ -0,0 +1,33 @@
+# https://editorconfig.org
+# Editor Config for WebMemex
+# Download an extension for editor config to allow your IDE
+# to follow the rules of the project
+
+# Used to indicate the file is in root of the project
+root = true
+
+# Styling applicable to all forms of files
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+end_of_line = lf
+charset = utf-8
+
+# Styling applicable to the file extension given
+[*.js, *.jsx]
+# This is due to the fact that github allows only 119 columns of space to view
+max_line_length = 119
```

Comment:
Actually this is a widely observed standard in many open source projects
```
max_line_length = 119
```
This is done so that there is no need for horizontal scrolling while reading the code as github provides only 119 columns worth of space to read the code. But I can change if it seems too restricting.